Sample Application/SampleApplication/Models/Authenticator.swift

```diff
@@ -0,0 +1,75 @@
+// 
+//  Authenticator.swift
+//  SampleApplication
+//
+//  Created by Okhan Okbay on 23/05/2023.
+//
+
+import CheckoutNetwork
+import Foundation
+
+#if canImport(CheckoutCardManagement)
+import CheckoutCardManagement
+#elseif canImport(CheckoutCardManagementStub)
+import CheckoutCardManagementStub
+#endif
+
+/*
+ Where documented, you are required to perform StrongCustomAuthentication (SCA)
+ in accordance to your company's policies and processes.
+ The sample project integrates directly with CKO backend for token creation in order to enable the flows,
+ but in a live application this is expected to occur from your backend to CKO backend and not be performed via your application.
+ Hence you can assess CheckoutNetwork as a network wrapper that you will not use.
+ */
+
+struct Authenticator {
+    private let networkClient: CheckoutNetworkClient = .init()
+    
+    func authenticateUser(checkDeviceOwnership: Bool = true,
+                          completionHandler: @escaping ((Result<String, CardManagementError>) -> Void)) {
+        
+        guard checkDeviceOwnership else {
+            authenticate(completionHandler: completionHandler)
+            return
+        }
+        
+        AuthenticationValidator.isDeviceOwner { isDeviceOwner in
+            guard isDeviceOwner else {
+                completionHandler(.failure(.authenticationFailure))
+                print("Please prove that you are the device owner." +
+                      "If you are on simulator, you can type anything and hit Enter")
```

Comment:
```suggestion
                print("SCA Failed! To pass make sure you provide any input to the check")
```

You don't really need more as if it runs on a real device it will look like a normal unlock screen, so presumably any device owner knows what to do when they see it 😄.

If on simulator, then console message should be sufficient to flag "hey, we know something went wrong, just add some input".

Reply:
In fairness seeing that an SCA failure leads to an error outcome due to inability to authenticate is a good thing, not a bug.